lib/asset-js.js

```diff
@@ -36,6 +37,7 @@ const PodiumAssetJs = class PodiumAssetJs {
         async = false,
         defer = false,
         type = 'default',
+        hint = '',
```

Comment:
Are we sure `hint` is the name we want to use. Feels a bit vague to me. Is there a word we can use which is more clear that this ties the script to the fallback route?